src/central-briefcase.rst

```diff
@@ -0,0 +1,48 @@
+.. spelling::
+
+  Migrator
+
+Central and Briefcase
+=========================
+
+:doc:`briefcase-intro` is a desktop application which provides access to the Central API from graphical and command-line interfaces. It can make it easier for non-developers to automate some aspects of data management. In particular, it is useful for migrating data from another ODK-compatible server to Central.
+
+Briefcase was created to provide features that the older ODK server, Aggregate, did not have. With Central, we encourage users to either :doc:`use data exports from the Central interface <central-submissions>` or to write scripts to interact directly with the :doc:`Central API <central-api>`. If you find yourself reaching for Briefcase a lot with Central, we would like to hear about what features you use it for `on the forum <https://forum.getodk.org/c/support/6>`_.
+
+.. warning::
+
+  Pulling and pushing media files is memory intensive. If you get a “Success with errors” status from Briefcase, we recommend increasing your Central server memory. If increasing memory is not an option, you can also :ref:`add swap <central-install-digital-ocean-swap>`.
+
+Migrating from another ODK-compatible server to Central
+--------------------------------------------------------
+
+In most contexts where an existing ODK-compatible server is in use, we recommend completing ongoing data collection projects before switching to Central. However, some long-running studies may want to make the switch before project completion. There may also be some contexts where storing data from past data collection projects in Central makes sense.
+
+ODK Briefcase can be used to perform a migration of both blank forms and form submissions to Central. Please be aware of the following limitations:
+
+* The submission date for all records will be the date that they were migrated to Central, not the date that they were submitted to the original server.
+* The submitter for all records will be the user that performs the migration. You may want to create a special user for this purpose and name it something like “Aggregate Migrator”.
```

Comment:
Migrator is kinda weird. I'd call it "Aggregate Migration" and remove the spelling thing up top.